py/nlraarch64.c

```diff
@@ -74,7 +74,7 @@ NORETURN void nlr_jump(void *val) {
         "ret                     \n"
         :
         : "r" (top)
-        :
+        : "x27", "x28", "x25", "x26", "x23", "x24", "x21", "x22", "x19", "x20", "x9"
```

Comment:
It would be nice to put these in numerical order. Also could add a comment to explain why x29 is omitted.

Reply:
Numerical would make sense, I just threw them in the order they appeared. X29 is omitted because as the frame pointer, it (a) won't be allocated to hold `top` by the compiler and (b) will throw a compile error (at least on clang) if listed as clobbered.

I'll fix it tomorrow morning.

Reply:
I've now incorporated those suggestions into the PR.